generator/collector_boundary_postcode.cpp

```diff
@@ -62,6 +67,28 @@ void BoundaryPostcodeCollector::Save()
     CHECK(base::CopyFileX(GetTmpFilename(), GetFilename()), ());
 }
 
+void BoundaryPostcodeCollector::OrderCollectedData()
+{
+  std::vector<std::pair<std::string, FeatureBuilder::PointSeq>> collectedData;
+  {
+    FileReader reader(GetFilename());
+    ReaderSource src(reader);
+    while (src.Size() > 0)
+    {
+      collectedData.resize(collectedData.size() + 1);
```

Comment:
вот кусочек из доки про resize():
If n is greater than the current container size, the content is expanded by inserting at the end as many elements as needed to reach a size of n. If val is specified, the new elements are initialized as copies of val, otherwise, they are value-initialized.
т.е. это работает точно так же как и через push_back

Reply:
++ решил тут поменять, через push_back({}) лучше читается

Reply:
этот кусочек ничего не исключает квадратичность, но она и правда явно исключена в доке:
```Linear in the difference between the current size and count.```

ок